src/app/page.tsx

```diff
@@ -37,7 +42,30 @@ export default async function Main({ searchParams }: { searchParams: { token: st
       <ClientSideStateUpdate workflowStates={workflowStates} tasks={tasks}>
         <DndWrapper>
           <Header showCreateTaskButton={true} />
-          <TaskBoard />
+          <TaskBoard
+            handleCreate={async (title, description, workflowStateId) => {
+              'use server'
+              fetch(`${apiUrl}/api/tasks?token=${token}`, {
+                method: 'POST',
+                body: JSON.stringify({
+                  title,
+                  body: description,
+                  workflowStateId,
+                }),
+              })
+              revalidateTag('getAllTasks')
+            }}
+            updateWorkflowStateIdOfTask={async (taskId, targetWorkflowStateId) => {
+              'use server'
+              fetch(`${apiUrl}/api/tasks/${taskId}?token=${token}`, {
+                method: 'PATCH',
+                body: JSON.stringify({
+                  workflowStateId: targetWorkflowStateId,
+                }),
+              })
+              revalidateTag('getAllTasks')
+            }}
+          />
```

Comment:
Please move this server action to a separate actions file

Reply:
bd5c271b2ef69e6ab2c0620d228e74a480d7ef37 and 89d3e104f611925474192ad4d6faea7e509b9c91 adds the actions in a separate file